Mathlib/Algebra/DirectSum/LinearMap.lean

```diff
@@ -4,6 +4,7 @@ Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Oliver Nash
 -/
 import Mathlib.Algebra.DirectSum.Module
+import Mathlib.LinearAlgebra.Eigenspace.Basic
```

Comment:
I note that the only file (currently) importing this one is `Algebra.Lie.Killing` which already depends on `LinearAlgebra.Eigenspace.Basic` (transitively).